.github/pull_request_template.md

```diff
@@ -5,7 +5,25 @@ Describe in plain language the motivation (bug, feature, etc.) behind the change
 Fixes: #NNN, #NNN, ...
 
 ## Release notes
+<!--
 
-If the PR changes the user experience, write a short summary of the changes. See the [CONTRIBUTING](https://github.com/vectorizedio/redpanda/blob/dev/CONTRIBUTING.md) guidelines for details.
+If this PR does not need to be included in the release notes, then
+simply have a bullet point for `none` directly under the `Release notes`
+section, e.g.
 
-Release note: [1-2 sentences of what this PR changes]
+* none
+
+Otherwise, add one or more of the following sections. A section must have
+at least 1 bullet point. You can add multiple sections with multiple
+bullet points if this PR represents multiple release note items. See
+the CONTRIBUTING.md guidelines for more details.
+
+### Features
```

Comment:
to be honest I am not sure how I would fill this out for something like rpk or k8s, could you try to explaining it a bit more in the template? I think if people won't understand, they won't do it. To me features and improvements sound like the same thing. And if it is a bugfix 🤷‍♀️

Reply:
what about "New features" and "Enhancements". with former being new capabilities of rpk/k8s, and the latter being changes to existing functionality. maybe @vsaraswat and team can chime in with a better delineation criteria

Reply:
@alenkacz i've created PR #3188 to explain what would be expected under the **Features** and **Improvements** sub-section with examples.

The **Bug fixes** section is not needed because it will be derived from instances of the `Fixes` keyword. Previous discussion about this approach: https://github.com/vectorizedio/redpanda/pull/2956#discussion_r759500134